projects/ad4170_iio/app/app_config.c

```diff
@@ -422,6 +420,9 @@ int32_t init_system(void)
 		return ret;
 	}
 
+	/* Delay to ensure that the peripherals are initialized */
+	no_os_mdelay(2000);
```

Comment:
nit: Are we facing any issue without this..? Is this also for the reconnection of SDP-K1 board?